assets/src/edit-story/components/canvas/element.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useLayoutEffect, useRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { getDefinitionForType } from '../../elements';
+
+const Wrapper = styled.div``;
+
+function Element( {
+	isEditing,
+	addNodeForElement,
+	setClickHandler,
+	handleSelectElement,
+	element: {
+		id,
+		type,
+		...rest
+	},
+} ) {
+	const { Display, Edit } = getDefinitionForType( type );
```

Comment:
Can there be an element that's not editable (or deep-editable as it might be)?

Reply:
Yes and in that case `Edit` will just be `undefined`. Elements only enter edit-mode, when they themselves request it based on user interaction.